### ISSUE TITLE
Combobox scrolling does not work

### ISSUE THREAD

**Issue by Tymur-Lohvynenko-Epicor** [vaadin/framework#12609]
Combobox popup scrolling does not work if `setScrollToSelectedItem(true)`, it's just stuck on one page.

Vaadin Framework version - 8.24.0

You can reproduce bug with the following code:
```java
ComboBox<String> comboBox = new ComboBox<>();
comboBox.setScrollToSelectedItem(true);

List<String> items = IntStream.range(1, 50)
        .mapToObj(n -> "Item #" + n)
        .collect(Collectors.toList());

comboBox.setItems(items);
comboBox.setValue(items.get(items.size() / 2));
```
Also you can use [repo-vaadin8-combobox-scrolling-bug](https://github.com/Tymur-Lohvynenko-Epicor/vaadin8-combobox-scrolling-bug) to reproduce this.

**Comment by TatuLund** [comment]
I tested this in my Vaadin 8.24 project with the code you have in this ticket, and I can't reproduce it. When I open ComboBox drop down, it scrolls to "Item #25" correctly.

I would like you to verify that when you have upgraded to Vaadin 8.24 you have recompiled the widgetset if you are having local widgetset. Misaligned widgetset is a common source for random glitches.

**Comment by TatuLund** [comment]
Furthermore we have very similar case included in our set of automated regression tests, this should prevent us releasing a version where this functionality would be trivially broken:

https://github.com/vaadin/framework/blob/master/uitest/src/main/java/com/vaadin/tests/components/combobox/ComboBoxScrollToSelectedItem.java

**Comment by Tymur-Lohvynenko-Epicor** [comment]
After testing with different versions I've found that this bug appeared in 8.18.0

**Comment by TatuLund** [comment]
Actually it is this PR that broke it in 8.18.1 https://github.com/vaadin/framework-es/pull/78

**Comment by rkovarik** [comment]
Hey 👋 

Disabling paging might serve as a potential workaround. However, this could lead to encountering the security limit for the number of items, which was introduced by https://github.com/vaadin/framework/pull/12415.
This could be avoided by increasing the limit https://github.com/vaadin/framework/pull/12466 but since we don’t know the actual size of the data, we would be forced to artificially increase the limit, potentially causing the security issue to reoccur.

We haven’t yet implemented the affected version in production, but this issue could hinder us from doing so. Is there a better workaround for this issue?

**Comment by thevaadinman** [comment]
The fix for this issue is due to be released with Vaadin 8.25.1.

**Comment by thevaadinman** [comment]
8.25.1 has been released.
